coursera_autograder/commands/__init__.py

```diff
@@ -2,7 +2,8 @@
 
 __all__ = [
     "upload",
-    "grade"
+    "grade",
+    "resources"
```

Comment:
I think we need to use "get_resource_limits" and "set_resource_limits"?

Reply:
It seems like it has to be "resources" here so main.py knows where to look for command parsers. I've changed the command itself to use `get_resources_limits` though.